extension/src/background.ts

```diff
@@ -66,14 +67,15 @@ export async function handlePasswordEntry(message: PasswordContent) {
   const url = message.url
   const host = getHostFromUrl(url)
   const password = message.password
+  const PROTECTEDROUTE = ProtectedRoutes[host as keyof typeof ProtectedRoutes]
```

Comment:
should it be uppercase for constants or camelcase?

Reply:
Yup fell off the consistency) Changed thanks for mentioning